Add matrix section of merged objects docs page (#229)
AlexsLemonade/scpca-docs: docs/sce_file_contents.md

### SingleCellExperiment expression counts

> The `counts` and `logcounts` assays of the `SingleCellExperiment` object for single-cell and single-nuclei experiments contains the primary RNA-seq expression data.

Reviewer: ```suggestion
The `counts` and `logcounts` assays of the `SingleCellExperiment` object for single-cell and single-nuclei experiments contain the primary RNA-seq expression data.
```

> The `counts` assay contains raw counts represented as integers, and the `logcounts` assay contains normalized (on a per-library basis) counts as described in {ref}`the data post-processing section <processing_information:processed gene expression data>`.

Reviewer: ```suggestion
The `counts` assay contains raw counts represented as integers, and the `logcounts` assay contains normalized counts as described in {ref}`the data post-processing section <processing_information:processed gene expression data>`.
```

> Both assays include reads aligned to both spliced and unspliced cDNA (see the section on {ref}`Post Alevin-fry processing <processing_information:post alevin-fry processing>`).

Reviewer: This makes me assume that the `logcounts` also have read counts, which isn't true. I would be more explicit. 
```suggestion
The `counts` assay includes reads aligned to both spliced and unspliced cDNA (see the section on {ref}`Post Alevin-fry processing <processing_information:post alevin-fry processing>`).
```

The data is stored as a sparse matrix, where each column represents a cell or droplet, and each row represents a gene.
The `counts` and `logcounts` assays can be accessed with the following R code:

```r
counts(sce) # counts matrix
logcounts(sce) # logcounts matrix
```

Column names are cell barcode sequences, and row names are Ensembl gene IDs.
These names can be accessed with the following R code:

```r
colnames(sce) # matrix column names
rownames(sce) # matrix row names
```

There is an additional `spliced` assay which contains the counts matrix with only reads from spliced cDNA:

```r
assay(sce, "spliced") # spliced-only counts matrix

> ```

Reviewer: Again, I would remove this.
